### PR TITLE
Update GDD.md

### PR DESCRIPTION
**Pull request by Gratein**
Changing the Asset part 
Adding Stage Unlock with fast affordable price

### CODE REVIEW

File: GDD.md [hunk 1]
```diff
@@ -136,14 +136,19 @@ Upgrade Level | Multiplier Increase | Cost
...
-- Tile map (forest)
-- Obstacle (forest) 
-
-### Sound
-- BGM (Forest)
-- CHAR VOICE (BOSS, CHAR)
-- SFX (COIN, GAME OVER)
+Stage Unlock
+•	Heat Resistant clothes [5000 coin] – Unlock volcano stage
+•	Oxygen clothes [5000 coin] – Unlock sea stage
```
Comment (mmgfrcs) [inline GDD.md]: Should be moved to Stages section above

File: GDD.md [hunk 2]
```diff
@@ -136,14 +136,19 @@ Upgrade Level | Multiplier Increase | Cost
...
+
+#Asset
+##Character 
+•	Amane Kisora-chan(FREE ver) credit to Sapphiart Inc  (http://www.sapphiart.co.jp/)
+•	Beast
+•	Tile map (forest)
+•	Obstacle (forest)
```
Comment (mmgfrcs) [inline GDD.md]: Change bullets (•) to dashes (-) and remove Tabs

File: GDD.md [hunk 3]
```diff
@@ -136,14 +136,19 @@ Upgrade Level | Multiplier Increase | Cost
...
+
+##Sound
+•	Ambience (Forest)
+•	CHAR VOICE (Beast, CHAR)
+•	SFX (COIN, GAME OVER)
```
Comment (mmgfrcs) [inline GDD.md]: Change bullets (•) to dashes (-) and remove Tabs

**Review by mmgfrcs — CHANGES_REQUESTED** [review]
Changes needed

File: GDD.md (hunk 2 above)
Comment (Gratein) [reply GDD.md]: roger

File: GDD.md (hunk 3 above)
Comment (Gratein) [reply GDD.md]: oke

File: GDD.md (hunk 1 above)
Comment (Gratein) [reply GDD.md]: okay

**Review by mmgfrcs — APPROVED** [review]
Tabs are still not removed yet. It is ignored by the renderer, so I'll approve this

**Pull request merged**